src/index.ts

```diff
@@ -1,6 +1,24 @@
+import './extensions';
 
-const say = (input: number) => input.toString(); 
 const results = Array.from({ length: 101 })
-                     .map((_, index) => `${index} => ${say(index)}`);
+                     .map((_, index) => `${index} => ${index.Fizzbuzz()}`);
 
 console.log(results.join('\n'));
+
+console.log("======");
+
+const contiguous = Array.from<number>({ length: 101 })
+            .map((_, index) => {
+                const convertValue = index.Fizzbuzz();
+                const previousConvertValue = (index-1).Fizzbuzz();
```

Comment:
contiguousness is checked against [-1..99] instead of [0..100] because of `index - 1`

Reply:
unit test ?